node/lib/util/submodule_rebase_util.js

```diff
@@ -209,12 +232,22 @@ exports.subConflictErrorMessage = function (name) {
     return `Conflict in ${colors.red(name)}`;
 };
 
+/**
+ * Log a message indicating that the specified `commit` is being applied.
+ *
+ * @param {NodeGit.Commit} commit
+ */
+exports.logCommit = function (commit) {
```

Comment:
nit: no space after function keyword?

Reply:
I always put a space after the function keyword.